docs/src/main/asciidoc/spring-cloud-config-client.adoc

```diff
@@ -0,0 +1,140 @@
+The end result of that process is a running Config Server that will provide the `Hello world` value for a configuration property named `message` when the application querying the server is named `a-bootiful-client`.
+
+== Creating the Maven project
+
+First, we need a new project. Create a new project with the following command:
+
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=spring-cloud-config-quickstart \
+    -DclassName="org.acme.spring.cloud.config.client.GreetingResource" \
+    -Dpath="/greeting" \
+    -Dextensions="spring-cloud-config-client"
+cd spring-cloud-config-quickstart
+----
+
+This command generates a Maven project with a REST endpoint and imports the `spring-cloud-config-client` extension.
+
+== GreetingController
+
+The Quarkus Maven plugin automatically generated a `GreetingResource` JAX-RS resource in the
+`src/main/java/org/acme/spring/cloud/config/client/GreetingResource.java` file that looks like:
+
+[source,java]
+----
+package org.acme.spring.spring.cloud.config.client;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return "hello";
+    }
+}
+----
+
+As we want to use configuration properties obtained from the Config Server, we will update the `GreetingResource` to inject the `message` property. The updated code will look like this:
+
+[source,java]
+----
+package org.acme.spring.spring.cloud.config.client;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @ConfigProperty(name = "message", defaultValue="hello default")
+    String message;
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return message;
+    }
+}
+----
+
+== Configuring the application
+
+Quarkus provides various configuration knobs under the `quarkus.spring-cloud-config` root. For the purposes of this guide, our Quarkus application is going to be configured in `application.properties` as follows:
+
+[source,properties]
+----
+# use the same name as the application name that was configured when standing up the Config Server
+quarkus.application.name=a-bootiful-client
+# enable retrieval of configuration from the Config Server - this is off by default
+quarkus.spring-cloud-config.enabled=true
+# configure the URI where the Config Server listens to HTTP requests - this could have been left out since http://localhost:8888 is the default
```

Comment:
```suggestion
# configure the URL where the Config Server listens to HTTP requests - this could have been left out since http://localhost:8888 is the default
```